ci/prepare_freebsd.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+echo "Preparing FreeBSD environment"
+sysctl hw.model hw.machine hw.ncpu
+set -e
+set -x
+
+env ASSUME_ALWAYS_YES=YES pkg bootstrap
+pkg install -y bash git cmake flex bison python3 ninja llvm11 zeek base64 ccache
+
+pyver=$(python3 -c 'import sys; print(f"py{sys.version_info[0]}{sys.version_info[1]}")')
+pkg install -y "$pyver"-pip
+pip install btest
+
+# The Zeek port is currently incomplete, get the missing files from a checkout.
+# The branch we pick here roughly matches the version of zeek packaged in above
+# port.
+git clone --branch release/zeek-3.0 https://github.com/zeek/cmake /usr/local/share/zeek/cmake
```

Comment:
This shouldn't be needed anymore as the distribution package now includes these files

Reply:
Looks like the package is still not built. I'd suggest we wait a couple more days for it to become available instead of building it ourself.

Reply:
The package is now available.